### PR TITLE
enhance-quickstart: Update description

### PR DESCRIPTION
**Pull request by satwikkansal**
* Remove redundant milestone links
* Update the status of the project

### CODE REVIEW

File: data/projects/enhance_coala_quickstart.md [hunk 1]
```diff
@@ -23,22 +23,16 @@ However, there’s scope for improvement in the following areas:
...
-* Figure out all the useful information that could be
+* *Figure out all the useful information that could be
   extracted from these files.
-* Get the cEP https://github.com/coala/cEPs/issues/80 merged.
+* *Get the cEP https://github.com/coala/cEPs/issues/80 merged.
 
 ##### [CODING PHASE 1: Extracting Information](https://gitlab.com/coala/GSoC-2017/milestones/20) 30th May - 26th June
 
-* The basic structure of the information extraction system is implemented.
+* **The basic structure of the information extraction system is implemented.
```
Comment (Adrianzatreanu) [inline data/projects/enhance_coala_quickstart.md]: why are you making them italic? due to the discussion we had with wip stuff?

**Comment by satwikkansal** [comment]
Yes, it's related to the discussion we had on previous PR. But actually I
had no intent of italicizing it, I'll check how it renders and will replace
asterisk with any other character if required.

**Comment by Adrianzatreanu** [comment]
i think that italicizing it may be a little overkill. however, me and my student had a better idea: checkboxes. what do you think? https://github.com/coala/projects/blob/master/data/projects/coala_cli.md check here and let me know your thoughts

**Comment by satwikkansal** [comment]
oh yeah, checkboxes make more sense actually.

**Comment by satwikkansal** [comment]
Switched to checkboxes :)

**Comment by Adrianzatreanu** [comment]
i like it

**Comment by Adrianzatreanu** [comment]
ack 237c2ed

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**